ProcessesApi/V1/Gateways/SoleToJointGateway.cs

```diff
@@ -51,101 +53,118 @@ private async Task<Person> GetPersonById(Guid id)
+        /// Passes if the tenant is marked as a named tenure holder (tenure type is tenant)
+        /// </summary>
+        private bool BR2(Guid tenantId, TenureInformation tenure)
         {
-            var paymentAgreements = await GetPaymentAgreementsByTenancyReference(tenureRef, Guid.NewGuid()).ConfigureAwait(false); // TODO: Confirm what correlation ID to use
-            if (paymentAgreements != null && paymentAgreements.Agreements.Count(x => x.Amount > 0) > 0)
-            {
-                return false;
-            }
-            else
-            {
-                var tenancy = await GetTenancyByReference(tenureRef, Guid.NewGuid()).ConfigureAwait(false); // TODO: Confirm what correlation ID to use
-                if (tenancy is null)
-                    return true;
+            var currentTenantDetails = tenure.HouseholdMembers.FirstOrDefault(x => x.Id == tenantId);
+            return currentTenantDetails.PersonTenureType == PersonTenureType.Tenant;
+        }
 
-                if (tenancy.nosp.active)
-                    return false;
+        /// <summary>
+        /// Passes if the tenant is not already part of a joint tenancy (there is not more than one responsible person)
+        /// </summary>
+        private bool BR3(TenureInformation tenure)
+        {
+            return tenure.HouseholdMembers.Count(x => x.IsResponsible) <= 1;
+        }
 
-                return true;
-            }
+        /// <summary>
+        /// Passes if the tenure is secure
+        /// </summary>
+        private bool BR4(TenureInformation tenure)
+        {
+            return tenure.TenureType.Code == TenureTypes.Secure.Code;
         }
 
-        private string GetLegacyTagRef(TenureInformation tenure)
+        /// <summary>
+        /// Passes if the tenure is active
+        /// </summary>
+        private bool BR6(TenureInformation tenure)
         {
-            var reference = tenure.LegacyReferences.FirstOrDefault(x => x.Name == "uh_tag_ref");
-            return reference?.Value;
+            return tenure.IsActive;
         }
 
-        public async Task<bool> CheckPersonTenureRecord(Guid tenureId, Guid proposedTenantId)
+        /// <summary>
+        /// Passes if there are no active payment agreeements on the tenure
+        /// </summary>
+        private async Task<bool> BR7(TenureInformation tenure)
         {
-            var tenure = await GetTenureById(tenureId).ConfigureAwait(false);
-            if (tenure is null)
-                return true; // skips this tenure if it doesn't exist
+            var tenancyRef = tenure.LegacyReferences.FirstOrDefault(x => x.Name == "uh_tag_ref");
 
-            var personHouseholdMemberRecord = tenure.HouseholdMembers.ToListOrEmpty().Find(x => x.Id == proposedTenantId);
-            if (personHouseholdMemberRecord is null)
-                return true; // skips this tenure if the person isn't listed as a household member
+            if (tenancyRef is null) return true;
```

Comment:
Should this be false here? My concern is that the new tenures will not have a UH tag ref, but that doesn't mean that there is no payment agreements/arrears. This check is the revisited once we have a confirmation for what the rule should be for BR7.